backend/dataall/core/environment/services/environment_service.py

```diff
@@ -341,6 +353,12 @@ def remove_group(session, uri, group):
             resource_uri=environment.environmentUri,
             resource_type=Environment.__name__,
         )
+        PolicyManager(
+            role_name=group_membership.environmentIAMRoleName,
+            environmentUri=environment.environmentUri,
+            account=environment.AwsAccountId,
+            resource_prefix=environment.resourcePrefix
+        ).delete_all_policies()
```

Comment:
Should these lines be moved to above L346-348 (before we delete the group_membership object)

Not sure if reference to `group_membership.environmentIAMRoleName` breaks after than but getting local IDE flag `"environmentIAMRoleName" is not a known member of "None"`